workbench-core/accounts/src/models/accountCfnTemplate.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+interface TemplateResponse{
+    createUrl: string;
+    updateUrl: string;
+}
+
+interface AccountCfnTemplateParameters{
+    accountHandlerRole : string;
+    apiHandlerRole : string;
+    enableFlowLogs : string;
```

Comment:
this is some interesting formatting. Is `rush prettier` being run as a pre-commit hook on your machine?

Reply:
Note: if not, I believe `rush cinstall` or `rush cupdate` will refresh your git hooks.